oam/install_scripts/mcs-loadbrm.py.in

```diff
@@ -98,7 +98,7 @@ if __name__ == '__main__':
         if use_systemd is True:
             cmd = 'systemctl start mcs-storagemanager'
             retcode = subprocess.call(cmd, shell=True)
-            if retcode < 0:
+            if retcode != 0:
```

Comment:
Just for reference. 
`subprocess.check_call`can be used inside try..catch block instead of using if condition.